src/commands/database/mysql/shell.tsx

```diff
@@ -12,12 +12,16 @@ import {
   withMySQLId,
 } from "../../../lib/database/mysql/flags.js";
 import { getConnectionDetailsWithPassword } from "../../../lib/database/mysql/connect.js";
+import { sshUsageDocumentation } from "../../../lib/ssh/doc.js";
 
 export class Shell extends ExecRenderBaseCommand<
   typeof Shell,
   Record<string, never>
 > {
   static summary = "Connect to a MySQL database via the MySQL shell";
+  static description =
+    "This command opens an interactive mysql shell to a MySQL database.\n\n" +
+    sshUsageDocumentation;
```

Comment:
should the new option be supported for this command as well? 

Currently the flag is not listed in the updated readme.

Reply:
Good point! :+1:  Hang on... ⏳

Reply:
Also fixed 🙆‍♂️